src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,210 @@
+
+package kitchenpos.application;
+
+import static kitchenpos.application.MenuGroupServiceFixture.menuGroup;
+import static kitchenpos.application.MenuServiceFixture.menu;
+import static kitchenpos.application.MenuServiceFixture.menus;
+import static kitchenpos.application.ProductServiceFixture.product;
+import static kitchenpos.application.ProductServiceFixture.products;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroupRepository;
+import kitchenpos.domain.MenuRepository;
+import kitchenpos.domain.Product;
+import kitchenpos.domain.ProductRepository;
+import kitchenpos.infra.PurgomalumClient;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+    @Mock
+    MenuRepository menuRepository;
+
+    @Mock
+    MenuGroupRepository menuGroupRepository;
+
+    @Mock
+    ProductRepository productRepository;
+
+    @Mock
+    PurgomalumClient purgomalumClient;
+
+    @InjectMocks
+    MenuService menuService;
+
+    private static final String FUCK = "fuck";
+
+    @DisplayName("메뉴를 등록할 수 있다.")
+    @Test
+    void create() {
+
+        //given
+        Menu menu = menu();
```

Comment:
```suggestion
        Menu menu = MenuServiceFixture.menu();
```
생략해도 되지만 명시적으로 표현해주는 것도 좋을 것 같네요 :)